src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -3044,6 +3044,129 @@ Many use cases can be fulfilled using this API. For instance:
   can scale up and down these resources when receiving the corresponding signals from users (via the Workflow Execution portal),
   or from third-party systems (via the Signals REST endpoint).
 
+== ProActive Modules APIs
+
+In addition to the core component APIs described above, ProActive workflow tasks can also interact with other ProActive Modules such as Service Automation and Notification Service.
+
+=== ProActive Service Automation (PSA) API
+
+The ProActive Service Automation API, or ProActive Service Automation Client, is a way by which a workflow can communicate with the link:../PSA/PSAUserGuide.html[Service Automation module]
+
+The ProActive Service Automation Client hides the complexity of building and sending HTTP requests manually by providing
+ready-to-use code callable from various tasks.
+
+Below is an example of a groovy script starting a managed service.
+
+NOTE: This example has been truncated for readability and is not meant to be used as is.
+
+[source,groovy]
+----
+// Import necessary classes from the Service Automation Client
+import org.ow2.proactive.pca.service.client.ApiClient
+import org.ow2.proactive.pca.service.client.api.ServiceInstanceRestApi
+import org.ow2.proactive.pca.service.client.model.ServiceInstanceData
+import org.ow2.proactive.pca.service.client.model.ServiceDescription
+import com.google.common.base.Strings;
+
+// Get session id using the Scheduler API
+def sessionId = schedulerapi.getSession()
+
+// Create a ServiceInstanceRestApi object
+def pcaUrl = variables.get('PA_CLOUD_AUTOMATION_REST_URL')
+def apiClient = new ApiClient()
+apiClient.setBasePath(pcaUrl)
+def serviceInstanceRestApi = new ServiceInstanceRestApi(apiClient)
+
+// Read workflow variables
+def serviceId = variables.get("SERVICE_ID")
+def instanceName = variables.get("INSTANCE_NAME")
+def serviceActivationWorkflow = variables.get("SERVICE_ACTIVATION_WORKFLOW")
+
+// Prepare service description
+ServiceDescription serviceDescription = new ServiceDescription()
+serviceDescription.setBucketName(bucketName)
+serviceDescription.setWorkflowName(startingWorkflowName)
+
+// Run service
+def serviceInstanceData = serviceInstanceRestApi.createRunningServiceInstance(sessionId, serviceDescription, Integer.parseInt(variables.get("PA_JOB_ID")))
+
+// Acquire service Instance ID
+def serviceInstanceId = serviceInstanceData.getInstanceId()
+
+// Acquire service endpoint
+serviceInstanceData = serviceInstanceRestApi.getServiceInstance(sessionId, serviceInstanceId)
+endpoint = serviceInstanceData.getDeployments().iterator().next().getEndpoint().getUrl()
+
+// Store variables and result
+variables.put("INSTANCE_ID_" + instanceName, serviceInstanceId)
+variables.put("ENDPOINT_" + instanceName, endpoint)
+result = endpoint
+----
+
+The complete ProActive Service Automation Client API is available in the javadoc link:../javadoc/cloudAutomationClient[Service Automation Client JavaDoc].
```

Comment:
don't you need a slash at the end of the url ?
link:../javadoc/cloudAutomationClient/
or eventually:
link:../javadoc/cloudAutomationClient/index.html

Reply:
Yes you are right. It seems like both a slash at the end or index.html works